packages/appstore/src/constants/wallet-description.ts

```diff
@@ -0,0 +1,86 @@
+    },
+    deriv_p2p: {
+        title: localize('Deriv P2P USD wallet'),
+        description: localize("Deposit and withdraw in Bitcoin, the world's first cryptocurrency."),
+        deposit_information: localize('Deposits are instant'),
+        withdrawal_information: localize('Withdrawals take 1 hour to 2 days'),
+    },
+    ethereum: {
+        title: localize('ETH wallet'),
+        description: localize(
+            'Quick, simple to use, and available anywhere, anytime. Used by millions of clients worldwide.'
+        ),
+        deposit_information: localize('Deposits are instant'),
+        withdrawal_information: localize('Withdrawals take 1 hour to 2 days'),
+    },
+    eur: {
+        title: localize('EUR wallet'),
+        description: localize(
+            'Quick, simple to use, and available anywhere, anytime. Used by millions of clients worldwide.'
+        ),
+        deposit_information: localize('Deposits are instant'),
+        withdrawal_information: localize('Withdrawals take 1 hour to 2 days'),
+    },
+    gbp: {
+        title: localize('GBP wallet'),
+        description: localize(
+            'Quick, simple to use, and available anywhere, anytime. Used by millions of clients worldwide.'
+        ),
+        deposit_information: localize('Deposits are instant'),
+        withdrawal_information: localize('Withdrawals take 1 hour to 2 days'),
+    },
+    litecoin: {
+        title: localize('LTC wallet'),
+        description: localize("Deposit and withdraw in Bitcoin, the world's first cryptocurrency."),
+        deposit_information: localize('Deposits are instant'),
+        withdrawal_information: localize('Withdrawals take 1 hour to 2 days'),
+    },
+    payment_agent: {
+        title: localize('Payment agent wallet'),
+        description: localize(
+            'Quick, simple to use, and available anywhere, anytime. Used by millions of clients worldwide'
+        ),
+        deposit_information: localize('Deposits are instant'),
+        withdrawal_information: localize('Withdrawals take 1 hour to 2 days'),
+    },
+    tether: {
+        title: localize('Tether wallet'),
+        description: localize(
+            'Quick, simple to use, and available anywhere, anytime. Used by millions of clients worldwide.'
+        ),
+        deposit_information: localize('Deposits are instant'),
+        withdrawal_information: localize('Withdrawals take 1 hour to 2 days'),
+    },
+    usd: {
+        title: localize('USD wallet'),
+        description: localize(
+            'Quick, simple to use, and available anywhere, anytime. Used by millions of clients worldwide.'
+        ),
+        deposit_information: localize('Deposits are instant'),
+        withdrawal_information: localize('Withdrawals take 1 hour to 2 days'),
+    },
+    usd_coin: {
+        title: localize('USD coin'),
+        description: localize("Deposit and withdraw in Bitcoin, the world's first cryptocurrency."),
```

Comment:
this description is not current for `usd_coin`

Reply:
@balakrishna-binary could not find many description in figma, so the values in the constants are not proper, I will update it for usd_coin but the constant values needs to be changed at one point